packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -123,7 +123,7 @@ export const DateInput: React.VFC<DateInputProps> = ({
       {...inputProps}
     />
 
-    {status && (
+    {status != "default" && (
```

Comment:
small nit, but I feel like we should use a strict equal `!==` here since it should always match to default as a string. It also help avoid type coercion unless there's a chance we want to have that there for safety?

Reply:
Yep, my bad